homeassistant/components/mazda/manifest.json

```diff
@@ -1,11 +1,9 @@
 {
   "domain": "mazda",
   "name": "Mazda Connected Services",
-  "codeowners": ["@bdr99"],
-  "config_flow": true,
+  "codeowners": [],
   "documentation": "https://www.home-assistant.io/integrations/mazda",
+  "integration_type": "system",
```

Comment:
I realize this is incorrect, but it prevents the integration from showing up in the "Add integration" dialog.

Reply:
Can we set `config_flow` to `False` instead?

Reply:
That is already the case. That would still make it appear (and send people to the docs to add it as a YAML-based integration).

Reply:
:see_no_evil: forgot that we changed how that is handled.